README.rst

```diff
@@ -267,23 +267,36 @@ mailing list. All interested parties are invited to follow the discussion.
    the discussion.
 
 -  Based on the proposal text (but not the GitHub commentary), the shepherd
-   makes a recommendation as to whether the proposal ought to be accepted,
-   rejected or returned for revision.
-
--  Discussion among the committee ensues on the mailing list.
-   Silence is understood as agreement with the shepherds recommendation.
+   decides whether the proposal ought to be accepted or rejected or returned for
+   revision.
+   
+-  If the shephard thinks the proposal ought to be rejected, they post their
+   justifications on the GitHub thread, and invites the authors to respond with
+   a rebuttal and/or refine the proposal. This continues until the either
+    
+   * the shepherd changes their mind and supports the proposal now,
+   * the authors withdraw their proposal,
+   * the authors indicate that they will revise the proposal to address the shepherds
+     point. The shepherd will label the pull request as
+     `Needs Revision <https://github.com/ghc-proposals/ghc-proposals/pulls?q=label%3A"Needs+revision">`_.
+   * the authors and the shepherd fully understand each other’s differing
+     positions, even if they disagree on the conclusion.
+
+-  Now the shepherd proposes acceptance or rejectance, by email to the mailing
```

Comment:
The committee discussion does sometimes uncover concerns the shepherd did not consider :) And in such a case, it would be unfortunate if the process turned into 

1. the shepherd raises concerns with the author,
2. the author addresses the concerns,
3. the committee raises other concerns,
4. the author addresses the new concerns,
5. the committee makes a decision.

I think we can collapse 1/2 and 3/4 into two steps: (1) the committee discusses and provides an initial response, (2) the author responds to concerns. In particular, we don't have to reach consensus to provide feedback, it seems perfectly reasonable for the initial response to be "the committee is divided on this proposal, specifically on the following concerns...".

Reply:
I don't quite see how @gridaphobe's tweak to the tweak leaves us in a different place than we are now -- except that we don't blast out the "reject" label so quickly and perhaps post back on GitHub sooner. Perhaps one way to avoid the unfortunate 5-step process is to have the shepherd post their concerns on both GitHub and the list in tandem, inviting other committee members to join in. I'm not sure we'll always get this right, though.

Reply:
The main goal of my tweak is to produce a faster feedback cycle. A few different people have raised the issue of delays on the committee's end, and I believe a decent chunk of that is due to stagnating debates within the committee. So my hope is that by collecting feedback from the committee as a whole and then allowing the author to respond, we can keep the discussions alive.

Reply:
I agree that my proposed tweak works well if the shepherd can foresee all complains, but less so when the committee discussion reveals flaws.

Having all discussions on Github would solve that.

Another take on it could be that the shephard acts like an advocate in both direction, representing the author towards the commttee, presenting and defending the proposal, but also representing the commitee towards the authors, relaying and discussing new concerns raised the by the committee as they come up, not just at the end of the discussion.